### PR TITLE
Add prow configs for cloud-ingress-operator

### PR DESCRIPTION
**Pull request by jewzaam**
For cloud-ingress-operator
* create basic jobs
* enable tide
* enable approval plugin
* set jenkins context

### CODE REVIEW

File: ci-operator/config/openshift/cloud-ingress-operator/openshift-cloud-ingress-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: ubi
+    namespace: ocp
+    tag: "8"
+binary_build_commands: unset GOFLAGS && make
```
Comment (cblecker) [inline ci-operator/config/openshift/cloud-ingress-operator/openshift-cloud-ingress-operator-master.yaml]:
```suggestion
```

I actually don't think we need this anymore.

**Comment by jewzaam** [comment]
/test generated-config
--

**Pull request merged**